djstripe/models.py

```diff
@@ -346,6 +346,16 @@ def add_card(self, source, set_default=True):
 
         return new_card
 
+    def upcoming_invoice(self, **kwargs):
+        """ Gets the upcoming preview invoice (singular) for this customer.
+
+        See the documentation in ``djstripe.stripe_objects.StripeInvoice.upcoming()` for details on method parameters.
+        The ``customer`` parameter is set to the customer object this method is being called on.
```

Comment:
The ``customer`` argument to the ``upcoming()`` call is automatically set by this method.